src/null-safety/index.md

```diff
@@ -176,24 +176,9 @@ environment:
 
 [language version]: /guides/language/evolution#language-versioning
 
-## Migrating existing code {#migrate}
```

Comment:
Since this is a historical document, I don't think you should remove this section. The page is about migrating, which still exists in this historical context.

Reply:
+1

Reply:
**comment:** It's not a historical document, but this should point to the migration guide instead of explaining anything here.